### ISSUE TITLE
Logic simplification and unnecessary loop elimination for csrf token

### ISSUE THREAD

**Issue by Christiandus** [vladimirs-git/fortigate-api#29]
Currently the [get token from cookies](https://github.com/vladimirs-git/fortigate-api/blob/401b2c7bde08aff49f57c97603a8ddf765421471/fortigate_api/fortigate_base.py#L212) function has the following loop:
<details>
  <summary>Current code</summary>

```
while True:
    # fortios < v7
    cookie_name = "ccsrftoken"
    if cookies := [o for o in session.cookies if o and o.name == cookie_name]:
        break

    # fortios >= v7
    cookie_name += "_"
    if cookies := [o for o in session.cookies if o and o.name.startswith(cookie_name)]:
        break

    raise ValueError("Invalid login credentials. Cookie 'ccsrftoken' is missing.")

token = str(cookies[0].value).strip('"')
return token
```
</details>  

In my opinion the logic could be significantly simplified and the loop removed. I've tested it on 7.2.8 and it works flawlessly :)
Let me know what you think!
(Of course the tests would need to be adjusted as they currently check for the underscore as well)  

<details>
  <summary>My suggestion</summary>

```
cookie_prefix = "ccsrftoken"
if cookies := [o for o in session.cookies if o and o.name.startswith(cookie_prefix)]:
    token = str(cookies[0].value).strip('"')
    return token
else:
    raise ValueError("Invalid login credentials. Cookie 'ccsrftoken' is missing.")
```
</details>

**Comment by vladimirs-git** [comment]
Thank you for your interest in this project. 
Regarding your suggestion to remove the following ValueError tests: [L103](https://github.com/vladimirs-git/fortigate-api/pull/28/files#diff-133f28fe8bca799932c5b8966d8ed887a3f408e94085ad05f6c380524740e37bL103) [L104](https://github.com/vladimirs-git/fortigate-api/pull/28/files#diff-133f28fe8bca799932c5b8966d8ed887a3f408e94085ad05f6c380524740e37bL104)

These tests aim to exclude cookies not related to authentication.
I suppose that some cookies starting with `ccsrftoken` might be intended for other purposes besides authentication and I prefer to do them to eliminate unpredictable behavior.
Perhaps my fears are unfounded.

**Comment by Christiandus** [comment]
When I checked the cookies set by the FGT there were only 2 cookies. The name of the other cookies starts with APSCOOKIE.
As you're only reading cookies from the FGT response (and not other websites) I don't see any problem in simplifying the logic, in the end it's up to you though.

**Comment by vladimirs-git** [comment]
fixed in [2.0.2](https://github.com/vladimirs-git/fortigate-api/releases/tag/2.0.2)
